acra/configuring-maintaining/general-configuration/acra_server.md

```diff
@@ -0,0 +1,419 @@
+---
+title: AcraServer
+bookCollapseSection: true
+weight: 3
+---
+
+# AcraServer
+
+## Command line flags
+
+* `--acracensor_config_file=<filename>`
+
+  Path to AcraCensor configuration file.
+
+* `--acraconnector_tls_transport_enable={true|false}`
+
+  Enable TLS to encrypt transport between AcraConnector and AcraServer.
+  Default is `false` which means "use SecureSession instead".
+
+* `--acraconnector_transport_encryption_disable=`
+
+  Use raw transport (tcp/unix socket) between AcraTranslator and client app.
+  It turns off reading trace from client app's side which usually sent by AcraConnector.
+  Default is `false`.
+
+* `--acrastruct_injectedcell_enable={true|false}`
+
+  Acrastruct may be injected into any place of data cell.
+  Default is `false`.
+
+* `--acrastruct_wholecell_enable={true|false}`
+
+  Acrastruct will stored in whole data cell.
+  Default is `true`.
+
+* `--audit_log_enable={true|false}`
+
+  Enable audit log functionality.
+  Default is `false`.
+
+* `--auth_keys=<filename>`
+
+  Path to basic auth credentials.
+  To add user, use: `./acra-authmanager --set --user <user> --pwd <pwd>`.
+  Default is `configs/auth.keys`.
+
+* `--client_id=<id>`
+
+  Use provided client ID for transparent encryption as if it was passed from AcraConnector.
+
+* `--db_host=<host>`
+
+  Database host for AcraServer -> database connections.
+  Must be set.
+  Default is empty.
+
+* `--db_port=<port>`
+
+  Database port for AcraServer -> database connections.
+  Default is `5432`.
+
+* `-ds`
+
+  Turn on HTTP debug server.
+  The server will be listening at `127.0.0.1:6060`.
```

Comment:
I'd move this to the “logging” section as it's more fitting to be there, as a debug tool.

Reply:
but it does not provide any logs. it is just an HTTP server for `go pprof` tool. not sure that it is obvious to search for such options in `logging` section... but we don't have a lot of options for debugging purposes to move them to a separate section. imho, we can leave it among other options for now.